### PR TITLE
fix: change radio button to checkbox for adding a new session

### LINKED ISSUE #4027: In session speaker form add a new session should be checkbox
**Describe the bug**
In session speaker form add a new session should be checkbox . Because once selecting a radio-button it is not possible to un-select it . So it should be checkbox as implemented in add a new speaker field.
In Add a newSession -
Screenshot from 2020-02-08 12-13-20
In add a new Speaker -
Screenshot from 2020-02-08 12-12-26

**Additional context**
working on it.

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4027 

captured (2)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/session-speaker-form.hbs [hunk 1]
```diff
@@ -332,9 +332,9 @@
       Or
     </div>
     <div class="field">
-      {{ui-radio name='sessionDetails' label=(t 'Add a new Proposal') value=false onChange=(action 'toggleNewSessionSelected' true)}}
+      {{ui-checkbox name='sessionDetails' class="toggle" label="Add a new proposal" checked=addNewSession onChange=(action (mut addNewSession))}}
```
Comment (kushthedude) [inline app/templates/components/forms/session-speaker-form.hbs]: Why renaming the actons ?

Comment (maze-runnar) [reply app/templates/components/forms/session-speaker-form.hbs]: if i will use (mut addNewsession) there will be no need of 
toggleNewSessionSelected(value) {
      this.set('sessionDetails', false);
      this.set('newSessionSelected', value);
    }
in session-form.js . So can i remove this action from session-speaker-form.js

Comment (maze-runnar) [reply app/templates/components/forms/session-speaker-form.hbs]: And also same action is not working correctly - 
captured (3)

Comment (kushthedude) [reply app/templates/components/forms/session-speaker-form.hbs]: Use class ui-checkbox with old actions. It should work .

Comment (maze-runnar) [reply app/templates/components/forms/session-speaker-form.hbs]: It's not working too - 
captured

**Pull request closed without merging**